### ISSUE TITLE
Tutorial for other species ?

### ISSUE THREAD

**Issue by forrestzhang** [igvteam/igv.js#241]
Hi, 

Is there any tutorial for using igv.js on other species, such as Arabidopsis? 

Thanks a lot!!

**Comment by jrobinso** [comment]
There is no tutorial but it would be straightforward to do.  What 
exactly is your question?  Have you browsed our documentation, 
specifically the section on defining your reference fasta.

**Comment by forrestzhang** [comment]
Thank for your reply, 

The code is my html file. I do not see any track and Chrome browser also do not show any error. 



`
<!DOCTYPE html>

<html lang="en">
<head>
    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="description" content="">
    <meta name="author" content="">
    <link rel="shortcut icon" href="//igv.org/web/img/favicon.ico">
    <title>Integrative Genomics Viewer </title>

    
    <link rel="stylesheet" type="text/css" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/css/bootstrap.min.css">

    
    <link rel="stylesheet" type="text/css"
          href="https://ajax.googleapis.com/ajax/libs/jqueryui/1.11.2/themes/smoothness/jquery-ui.css"/>

    
    <link rel="stylesheet" type="text/css"
          href="https://maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css"/>

    
    <link rel="stylesheet" type="text/css" href="https://igv.org/web/release/1.0.1/igv-1.0.1.css">

    
    <link rel="stylesheet" type="text/css" href="css/bam.css">

    
    <script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
    <script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jqueryui/1.11.2/jquery-ui.min.js"></script>

    
    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/js/bootstrap.min.js"></script>

    
    <script type="text/javascript" src="https://igv.org/web/release/1.0.1/igv-1.0.1.js"></script>


</head>

<body>

<div class="jumbotron">

    <div class="container">
        <h2>IGV.js embedded in a Bootstrap page.</h2>
    </div>
    

</div>


<div class="container-fluid" id="igvDiv" style="padding:5px; border:1px solid lightgray"></div>


<script type="text/javascript">

    $(document).ready(function () {

        var div = $("#igvDiv")[0],

                options = {
                  showNavigation: true,
                  showRuler: true,
                  genome: "tair10",
                  locus: "Chr2:8,997,292-8,997,392",
                  reference: {
                      id: "tair10",
                      fastaURL: "//jianglab.horticulture.wisc.edu/Download/Data/tair10.fa"
                      indexURL: "//jianglab.horticulture.wisc.edu/Download/Data/tair10.fa.fai"
                  },
                  tracks: [
                      {
                          name: "Genes",
                          url: "http://jianglab.horticulture.wisc.edu/Download/Data/Araport11_genes.20150701.gff3",
                          displayMode: "EXPANDED"
                      }
                  ]
              };

        igv.createBrowser(div, options);

    });

</script>

</body>

</html>

`

**Comment by forrestzhang** [comment]
I add reference part, but does't work.


```
<!DOCTYPE html>

<html lang="en">
<head>
    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="description" content="">
    <meta name="author" content="">
    <link rel="shortcut icon" href="//igv.org/web/img/favicon.ico">
    <title>Integrative Genomics Viewer </title>

    <!-- Bootstrap CSS - for demo only, NOT REQUIRED FOR IGV -->
    <link rel="stylesheet" type="text/css" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/css/bootstrap.min.css">

    <!-- jQuery UI CSS -->
    <link rel="stylesheet" type="text/css"
          href="https://ajax.googleapis.com/ajax/libs/jqueryui/1.11.2/themes/smoothness/jquery-ui.css"/>

    <!-- Font Awesome CSS -->
    <link rel="stylesheet" type="text/css"
          href="https://maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css"/>

    <!-- IGV CSS -->
    <link rel="stylesheet" type="text/css" href="https://igv.org/web/release/1.0.1/igv-1.0.1.css">

    <!-- bam.css n - for demo only, NOT REQUIRED FOR IGV -->
    <link rel="stylesheet" type="text/css" href="css/bam.css">

    <!-- jQuery JS -->
    <script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
    <script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jqueryui/1.11.2/jquery-ui.min.js"></script>

    <!-- Bootstrap JS - for demo only, NOT REQUIRED FOR IGV -->
    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/js/bootstrap.min.js"></script>

    <!-- IGV JS-->
    <script type="text/javascript" src="https://igv.org/web/release/1.0.1/igv-1.0.1.js"></script>


</head>

<body>

<div class="jumbotron">

    <div class="container">
        <h2>IGV.js embedded in a Bootstrap page.</h2>
    </div>
    <!-- container -->

</div>
<!-- jumbotron -->

<div class="container-fluid" id="igvDiv" style="padding:5px; border:1px solid lightgray"></div>


<script type="text/javascript">

    $(document).ready(function () {

        var div = $("#igvDiv")[0],

                options = {
                  showNavigation: true,
                  showRuler: true,
                  genome: "tair10",
                  locus: "Chr2:8,997,292-8,997,392",
                  reference: {
                      id: "tair10",
                      fastaURL: "http://jianglab.horticulture.wisc.edu/Download/Data/tair10.fa",
                      indexURL: "http://jianglab.horticulture.wisc.edu/Download/Data/tair10.fa.fai"
                  },
                  tracks: [
                      {
                          name: "Genes",
                          url: "http://jianglab.horticulture.wisc.edu/Download/Data/Araport11_genes.20150701.gff3",
                          displayMode: "EXPANDED"
                      }
                  ]
              };

        igv.createBrowser(div, options);

    });

</script>

</body>

</html>


```

![image](https://cloud.githubusercontent.com/assets/364708/17820583/1d9d3446-6613-11e6-8e0d-14023ded0eb0.png)

**Comment by jrobinso** [comment]
This might be a CORS problem if your web page has a different host than the data you are trying to load you have to add "CORS" headers.  This is a general rule of the web, nothing igv specific here.  You can read about "CORS" here:  https://developer.mozilla.org/en-US/docs/Web/HTTP/Access_control_CORS.    If this is an apache server you can have it add the required headers in an ".htaccess" file.

You can avoid CORS issues by serving your page from the same host as the data, i.e. from http://jianglab.horticulture.wisc.edu.   This makes development difficult, however, as you can't run pages from localhost.

**Comment by forrestzhang** [comment]
Thanks a lot. It's work now,

![image](https://cloud.githubusercontent.com/assets/364708/17821611/1ea3d0b6-6618-11e6-9718-7cd1eab63326.png)


```
options = {
                  showNavigation: true,
                  showRuler: true,
                  genome: "tair10",
                  locus: "Chr2:8,997,292-8,997,392",
                  reference: {
                      id: "tair10",
                      fastaURL: "/Data/tair10.fa",
                      indexURL: "/Data/tair10.fa.fai"
                  },
                  tracks: [
                      {
                          name: "Genes",
                          url: "/Data/Araport11_genes.20150701.gff3",
                          displayMode: "EXPANDED"
                      },
                      {
                        url: "/Data/Ath_leaf_DNase.bw",
                        type: "wig",
                        name: 'Leaf DHSs Score'
                      }

                  ]
              };
```

**Comment by jrobinso** [comment]
Great, looks good.
